################################################################################
SHORT DESCRIPTION: 
################################################################################
Allow IPXE to be built outside the Xen tree.

################################################################################
LONG DESCRIPTION: 
################################################################################
Pass the IPXE ROM, that is built out of Xen tree, path to the build system so it
can be embedded in hvmloader. This is an OpenXT build requirement.

################################################################################
CHANGELOG 
################################################################################

################################################################################
REMOVAL 
################################################################################

################################################################################
UPSTREAM PLAN
################################################################################
No.

################################################################################
INTERNAL DEPENDENCIES 
################################################################################

################################################################################
PATCHES 
################################################################################
Index: xen-4.6.5/tools/firmware/hvmloader/Makefile
===================================================================
--- xen-4.6.5.orig/tools/firmware/hvmloader/Makefile
+++ xen-4.6.5/tools/firmware/hvmloader/Makefile
@@ -59,6 +59,11 @@ endif
 
 ROMS := 
 
+ifneq ($(PXE_ROM),)
+ETHERBOOT_ROMS := $(PXE_ROM)
+ROMS += $(PXE_ROM)
+endif
+
 ifeq ($(CONFIG_OVMF),y)
 OBJS += ovmf.o
 CFLAGS += -DENABLE_OVMF
